contracts/utils/deploy-l2.js

```diff
@@ -256,9 +265,26 @@ function deployOnSonic(opts, fn) {
   return main;
 }
 
+function deployOnSonicWithAdmin(opts, fn) {
```

Comment:
I think we can just have `deployOnSonic` that can handle both Timelock actions and no actions.

Reply:
agreed